superset/assets/src/explore/controls.jsx

```diff
@@ -1561,6 +1561,7 @@ export const controls = {
       '1 week',
       '28 days',
       '30 days',
+      '364 days',
```

Comment:
Is 52 weeks clearer than 364 days?

Reply:
I think 364 days makes it clear that 1 year is 365 days.

Reply:
I also think 52 weeks conveys the "same day of week" idea better.